azure/services/managedclusters/spec.go

```diff
@@ -329,8 +334,10 @@ func (s *ManagedClusterSpec) Parameters(existing interface{}) (params interface{
 
 		diff := computeDiffOfNormalizedClusters(managedCluster, existingMC)
 		if diff == "" {
+			log.V(4).Info("no changes found between user-updated spec and existing spec")
 			return nil, nil
 		}
+		log.V(4).Info("spec has been updated", "difference", diff)
```

Comment:
nit: at that point nothing has been updated yet (diff could come from the Azure resource itself getting changed by the user outside of CAPZ). How about:

```suggestion
		log.V(4).Info("found a diff between the desired spec and the existing managed cluster", "difference", diff)
```

?

Reply:
Good thought! adding it

Reply:
Added via bb36e183ad2061a5f77073222e83ff0a54aa085d